### DISCUSSION TITLE
What Feature Should Demolition Engine Have PART 2?

### DISCUSSION THREAD

**Discussion by RealDealJinx** [DEMOLITIONDON96/Demolition-Engine#54] [Polls]
OK LAST ONE

**Comment by Theoyeah** [comment]
i've got something thats make the opponent gain health by pressing notes
tell me if you want i add it

**Reply by ThatOneSillyGuyDon** [reply]
> i've got something thats make the opponent gain health by pressing notes tell me if you want i add it

healthDrain code already exists in the files, just paste some existing code of the opponent draining your health lol

healthDrain code is used for the Poison Notes btw
